components/Trade/Advanced/TradingPanel/InterfaceSelect/index.tsx

```diff
@@ -35,7 +34,7 @@ type ISProps = {
 export default styled(({ className, isAdjust, setAdjust }: ISProps) => {
     return (
         <div className={className}>
-            <p>{isAdjust ? 'Adjust Position' : 'Place Order'}</p>
+            <p className="flex items-center">{isAdjust ? 'Adjust Position' : 'Place Order'}</p>
```

Comment:
Maybe add a `whitespace-nowrap` as I noticed the text wrap when you resize before getting down to mobile and perhaps a little right margin as well.